### PR TITLE
Break out common Make operations to standard.mk

### PR DESCRIPTION
**Pull request by lisa**
The goal is for this to be distributed to keep Makefiles predictable.
OSD Operator Makefiles need to have:

```
SHELL := /usr/bin/env bash

# Include shared Makefiles
include project.mk
include standard.mk

default: gobuild
```

This will provide the targets and variables the CI pipeline expects.
Extending the Makefile beyond that is available.

Signed-off-by: Lisa Seelye <lseelye@redhat.com>

### CODE REVIEW

**Comment by lisa** [comment]
#31 will introduce a merge conflict.

File: standard.mk [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+ifndef VERSION_MAJOR
+$(error VERSION_MAJOR is not set; check project.mk file)
+endif
+ifndef VERSION_MINOR
+$(error VERSION_MINOR is not set; check project.mk file)
+endif
+
+# Generate version and tag information from inputs
+COMMIT_NUMBER=$(shell git rev-list `git rev-list --parents HEAD | egrep "^[a-f0-9]{40}$$"`..HEAD --count)
+BUILD_DATE=$(shell date -u +%Y-%m-%d)
```
Comment (jewzaam) [inline standard.mk]: `BUILD_DATE` was dropped out of `master` along with other changes.  Looks like this file is based off an older commit.  Please reconcile against current `master` branch.

**Comment by lisa** [comment]
@jewzaam I think the rebase caught all the changes. How's this look?

File: standard.mk [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+.PHONY: test
+test: envtest gotest
+
+.PHONY: env
+.SILENT: env
+env: isclean
+	echo OPERATOR_NAME=$(OPERATOR_NAME)
+	echo OPERATOR_NAMESPACE=$(OPERATOR_NAMESPACE)
+	echo OPERATOR_VERSION=$(VERSION_FULL)
```
Comment (jewzaam) [inline standard.mk]:
```suggestion
	echo OPERATOR_VERSION=$(OPERATOR_VERSION)
```

**Pull request merged**